code/modules/cargo/packs/spacesuit_armor.dm

```diff
@@ -141,7 +141,7 @@
 	desc = "One set of well-rounded, tactical body armor. The set includes a helmet and vest."
 	cost = 1500
```

Comment:
```suggestion
	cost = 3000
```

Reply:
i think 3000 is too high for an armored vest upgrade (750)

Reply:
also im not doing that in this pr because im doing it in #3781